tests/test_config.py

```diff
@@ -21,6 +21,7 @@ def test_loading_empty(set_mlem_project_root):
     assert isinstance(config.storage, LocalStorage)
 
 
+@need_aws_auth
 @long
 def test_loading_remote(s3_tmp_path, s3_storage_fs):
     project = s3_tmp_path("remote_conf")
```

Comment:
Looks good, thank you @aminalaee. This should make it easier contributing 🙏🏻  The only question - I thought some tests with aws will pass since to **read** test bucket, you don't need any special permissions. Is this wrong?

Reply:
Not sure the case, but `boto3.Session().get_credentials()` will return None if the AWS credentials `~/.aws/credentials` doesn't exist on my local. I guess this is even before making the calls to AWS services.